packages/docs/src/components/DocsHeadline.vue

```diff
@@ -22,11 +26,11 @@ const PropsMixin = Vue.with(Props)
 })
 export default class DocsHeadline extends mixins(PropsMixin) {
   get textComputed () {
-    return this.$t(this.text)
+    return (this as any).$tie(this.text)
```

Comment:
`this as any`? is it not defined in the plugin?

Reply:
I have corrected (added type declaration file)